tests/nightly/models/test_model_nightly.py

```diff
@@ -42,14 +42,15 @@ def get_model_nncf_cat() -> List:
         List: Returns a combination of models with their nncf support for each category.
     """
     model_support = [
-        ("padim", False),
+        ("cflow", False),
         ("dfkde", False),
         ("dfm", False),
-        ("stfpm", False),
+        ("fastflow", False),
```

Comment:
I don't think this will work as the performance thresholds for fastflow haven't been updated in the performance threshold file.